R/readepi_helpers.R

```diff
@@ -130,3 +128,40 @@ fingertips_get_args <- function(args_list =
     parent_area_type_id = parent_area_type_id
   )
 }
+
+#' Check if the base_url argument is specified as a correct URL
+#'
+#' @param base_url the URL to the HIS of interest
+#'
+#' @keywords internal
+#'
+url_check <- function(base_url) {
+  checkmate::assert_character(base_url, any.missing = FALSE, len = 1L,
+                              null.ok = FALSE)
+  if (startsWith(base_url, "https?://")) {
```

Comment:
```suggestion
  if (grepl("^https?://", base_url)) {
```

I overlooked the fact that `startsWith()` doesn't work with regexes in my previous comment